gadgets/snapshot_socket/README.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: snapshot_socket
-sidebar_position: 35
+sidebar_position: 40
```

Comment:
The idea was to use multiples of 10 for the existing gadgets and then start filling the spaces with the new ones. But I suppose this approach of changing the position for all of them works as well. I'm wondering if the script you included in the commit message should be a makefile target.